### PR TITLE
Describe why apikey is needed, where to get them and the format.

### LINKED ISSUE #93: GitHub API rate limiting
The GitHub API is used anonymously in this library. If used more than [60 req per hour](https://docs.github.com/en/rest/overview/resources-in-the-rest-api#rate-limiting) the API will say no. This limit could be reached when asserting a bunch of software like on https://www.research-software.nl/ .

One way to solve is to use an [OAuth token](https://docs.github.com/en/rest/overview/resources-in-the-rest-api#oauth2-token-sent-in-a-header) which will up the limit to 5000 req per hour. It would be nice if the howfairis tool would allow passing a GitHub OAuth token either as environment variable or as command line argument.

GitLab also has [rate limit](https://docs.gitlab.com/ee/user/gitlab_com/#gitlabcom-specific-rate-limits), but is much higher so not such a problem.

### LINKED ISSUE #319: describe how to get api keys
see https://github.com/fair-software/howfairis/pull/286

### PR DESCRIPTION
**Pull request by sverhoeven**
Fixes #319
Fixes #93 completely

To review set and use env vars as described.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -319,6 +319,23 @@ The manual override will be reflected in the output, as follows:
...
+To increase the rate limit you need to use authenticated requests.
+Your username and token can be passed to ``howfairis`` using environment variables called ``APIKEY_GITHUB`` and ``APIKEY_GITLAB``.
+The format of the environment variable values are:
+
+.. code-block:: shell
+
+  export APIKEY_GITHUB=<user who made the token>:<personal access token>
+  export APIKEY_GITLAB=<user who made the token>:<personal access token>
+
+Your personal access token should be created on `GitHub <https://github.com/settings/tokens>`_ and `GitLab <https://gitlab.com/-/profile/personal_access_tokens>`_ settings pages.
```
Comment (fdiblen) [inline README.rst:336]: Will be better to use the link below for GitHub:
https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token

Comment (fdiblen) [reply README.rst:336]: And this is for Gitlab:
https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html#creating-a-personal-access-token

Comment (fdiblen) [inline README.rst:336]:
```suggestion
Your personal access token should be created on `GitHub <https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token
>`_ and `GitLab <https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html#creating-a-personal-access-token
>`_ settings pages.
```

Comment (sverhoeven) [reply README.rst:336]: good idea

**Review by fdiblen — APPROVED** [review]
Very clear text.. Thank you @sverhoeven :pray:

**Pull request merged**